mne_bids/write.py

```diff
@@ -777,9 +777,10 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
                    event_id=None, overwrite=False, verbose=True):
     """Walk over a folder of files and create BIDS compatible folder.
 
-    .. warning:: The original files are simply copied over. This function
-                 cannot convert modify data files from one format to another.
-                 Modification of the original data files is not allowed.
+    .. warning:: The original files are simply copied over if the original
+                 file format is BIDS-supported for that modality. Otherwise,
+                 this function will convert to a BIDS-supported file format
```

Comment:
Might be nice to be more specific. Say that you convert to brainvision format for EEG (and iEEG?) data.